lbrynet/core/BlobManager.py

```diff
@@ -1,21 +1,17 @@
 import logging
 import os
-import time
-import sqlite3
 
 from twisted.internet import threads, defer, reactor
-from twisted.enterprise import adbapi
 from lbrynet import conf
 from lbrynet.blob.blob_file import BlobFile
 from lbrynet.blob.creator import BlobFileCreator
 from lbrynet.core.server.DHTHashAnnouncer import DHTHashSupplier
-from lbrynet.core.sqlite_helpers import rerun_if_locked
 
 log = logging.getLogger(__name__)
 
 
 class DiskBlobManager(DHTHashSupplier):
-    def __init__(self, hash_announcer, blob_dir, db_dir):
+    def __init__(self, hash_announcer, blob_dir, storage):
```

Comment:
is it possible to require a class or interface for arguments? i feel like `storage` should be an interface with functions like "addBlobs", "getBlobs", etc. Then we can implement multiple types of storage (e.g. `sqlite` for production, `in-memory` for testing) and it's clear what the contract is.

A simple way of doing this now without too much refactoring is to create an interface that has all the functions that are already in use. Then we can pare it down later.